lib/Echidna/Types/Tx.hs

```diff
@@ -23,6 +23,12 @@ data TxCall = SolCreate   ByteString
 makePrisms ''TxCall
 $(deriveJSON defaultOptions ''TxCall)
 
+initialTimestamp :: Word
```

Comment:
i think it is better to eventually make these configurable but we'll leave it as a constant for now